docs/Deployment/BackupRestore.md

```diff
@@ -0,0 +1,324 @@
+
+```
+🚨  Decryption failed. Please check the error message above.
+```
+
+When the restore has been completed, you get the message : 
+
+```
+✅ Restore done
+```
+
+Everything should be up and running again when the restore is complete, and you've successfully restored your BTCpay Server.
+Congratulations!
+
+:::tip
+We won't nail every aspect right away with this approach. This page will be subject to change. 
+Always make sure your backup strategy is tested and fits your business needs. No one solution fits all.
+For the latest updates, always feel free to ask on the BTCPay Server community channels. 
+:::
+
+
+### Restore output
+
+`btcpay-restore.sh` Output example : 
+
+```
+ℹ️  Cleaning restore directory /var/lib/docker/volumes/backup_datadir/_data/restore …
+
+🔐 Decrypting backup file …
+gpg: AES256.CFB encrypted data
+gpg: encrypted with 1 passphrase
+✅ Decryption done.
+
+ℹ️  Extracting files in /var/lib/docker/volumes/backup_datadir/_data/restore…
+postgres.sql.gz
+volumes/generated_bitcoin_wallet_datadir/
+volumes/generated_bitcoin_wallet_datadir/_data/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/.walletlock
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/wallet.dat
+volumes/generated_bitcoin_wallet_datadir/_data/mainnet/db.log
+volumes/generated_btcpay_datadir/
+volumes/generated_btcpay_datadir/_data/
+volumes/generated_btcpay_datadir/_data/FastSynced
+volumes/generated_btcpay_datadir/_data/LocalStorage/
+volumes/generated_btcpay_datadir/_data/LocalStorage/tmp/
+volumes/generated_btcpay_datadir/_data/btcpay20220512.log
+volumes/generated_btcpay_datadir/_data/tmp/
+volumes/generated_btcpay_datadir/_data/host_id_rsa
+volumes/generated_btcpay_datadir/_data/Main/
+volumes/generated_btcpay_datadir/_data/Main/settings.config
+volumes/generated_btcpay_datadir/_data/host_id_rsa.pub
+volumes/generated_btcpay_datadir/_data/host_authorized_keys
+volumes/generated_btcpay_datadir/_data/key-47ec2b82-ed01-45ef-8a63-390470cc10e5.xml
+[...]
+volumes/generated_tor_torrcdir/
+volumes/generated_tor_torrcdir/_data/
+volumes/generated_tor_torrcdir/_data/torrc
+volumes/generated_tor_torrcdir/_data/torrc-2
+
+ℹ️  Restoring database …--
+-- PostgreSQL database cluster dump
+--
+SET default_transaction_read_only = off;
+SET
+SET client_encoding = 'UTF8';
+SET
+SET standard_conforming_strings = on;
+SET
+--
+-- Drop databases (except Postgres and template1)
+--
+DROP DATABASE btcpayservermainnet;
+DROP DATABASE nbxplorermainnet;
+ERROR:  database "btcpayservermainnet" is being accessed by other users
+DETAIL:  There is 1 other session using the database.
+
+[...]
+
+--
+-- PostgreSQL database dump complete
+--
+--
+-- PostgreSQL database cluster dump complete
+--
+
+ℹ️  Cleaning up …
+
+✅ Restore done
+```
+
+## What isn't included and should be considered before taking this backup approach. 
```

Comment:
Maybe introduce a seperate FAQ heading for these "question" type headings?

Reply:
Updated that part.